bittensor/utils/async_substrate_interface.py

```diff
@@ -768,7 +769,9 @@ def __init__(
         chain_name=None,
     ):
         """
-        The asyncio-compatible version of the subtensor interface commands we use in bittensor
+        The asyncio-compatible version of the subtensor interface commands we use in bittensor. It is important to
+        initialise this class asynchronously in an async context manager using `async with AsyncSubstrateInterface()`.
+        Otherwise, some (most) methods will not work properly, and may raise exceptions.
```

Comment:
plz add in this docstring all arguments since this class is considering as main class in this module